src/app/settings/llm/LLM/Checker.tsx

```diff
@@ -69,14 +65,17 @@ const Checker = memo<CheckerProps>(({ checkModel }) => {
             {t('llm.OpenAI.check.pass')}
           </Flexbox>
         )}
+        <Button loading={loading} onClick={checkConnection}>
+          {t('llm.OpenAI.check.button')}
+        </Button>
       </Flexbox>
 
       {error && (
-        <Flexbox gap={8}>
+        <Flexbox gap={8} width={'35vw'}>
```

Comment:
why this?

Reply:
因为错误内容展示的 div 宽度，也会因为具体的错误内容展开和关闭而变化产生抖动。所以我觉得应该固定个合适的宽度，也因为设备宽度的差异性，我使用了 `vw` 单位。
Because the width of the div displaying incorrect content will also change and cause jittering when the specific incorrect content is expanded or collapsed. So I think a suitable width should be fixed, and because of the differences in device widths, I used `vw` units.

Reply:
可能要确认下移动端上的表现

Reply:
嗷！还有移动端啊，那目前的改动确实存在问题。我再改改。

Reply:
> 可能要确认下移动端上的表现

已推送兼容移动端的修改，并更新演示视频。